client/src/features/searchV2/components/SearchV2Results.tsx

```diff
@@ -274,10 +274,15 @@ function SearchV2ResultProject({ project }: SearchV2ResultProjectProps) {
       : generatePath(ABSOLUTE_ROUTES.v2.groups.show.root, {
           slug: namespace?.namespace ?? "",
         });
-  const projectUrl = generatePath(ABSOLUTE_ROUTES.v2.projects.show.root, {
-    namespace: namespace?.namespace ?? "",
-    slug,
-  });
+  const projectUrl =
+    namespace?.namespace != null
+      ? generatePath(ABSOLUTE_ROUTES.v2.projects.show.root, {
+          namespace: namespace?.namespace ?? "",
```

Comment:
I think this should be correct since you test `namespace?.namespace != null`:

```suggestion
          namespace: namespace.namespace,
```